README.md

```diff
@@ -188,6 +189,7 @@
 [meetings-sidepanel#cs]:samples/meetings-sidepanel/csharp
 [meetings-sidepanel#js]:samples/meetings-sidepanel/nodejs
 [meetings-content-bubble#js]:samples/meetings-content-bubble/nodejs
+[app-in-meeting#cs]:samples/app-in-meeting/csharp
```

Comment:
Looks like we haven't added link for nodejs sample for app-in-meeting, please check & update if not added.